packages/s/scx-scheds/package.yml

```diff
@@ -0,0 +1,43 @@
+name       : scx-scheds
+version    : 1.0.16
+release    : 1
+source     :
+    - https://github.com/sched-ext/scx/archive/refs/tags/v1.0.16.tar.gz : f788d95aa66dbd30744c062e23ac71e55209016f951f74de9a3b214b583d82ff
+homepage   : https://github.com/sched-ext/scx
+license    : MIT
+component  : system.utils
+summary    : sched_ext schedulers and tools
+description: |
+    sched_ext schedulers and tools
+clang      : yes
+networking : yes
+builddeps  :
+    - pkgconfig(libbpf)
+    - pkgconfig(libelf)
+    - pkgconfig(libjq)
+    - pkgconfig(libseccomp)
+    - pkgconfig(libsystemd)
+    - pkgconfig(protobuf)
+    - linux-tools-bpftool
+    - rust-devel
+environment: |
+    # sscache is broken for this build. Disable it for now.
+    unset RUSTC_WRAPPER
+setup      : |
+    %meson_configure -Dbpftool=disabled \
+                     -Dlibbpf_a=disabled \
+                     -Dvendordir=/usr/share/defaults/
+    %cargo_fetch
+build      : |
+    %ninja_build
+install    : |
+    %ninja_install
+
+    # Unsupported, use scx_loader.
+    rm -v $installdir/usr/lib/systemd/system/scx.service
+    rm -rfv $installdir/etc/
+
+    # Use custom config.
+    install -Dm00644 $pkgfiles/config.toml $installdir/usr/share/defaults/scx_loader/config.toml
```

Comment:
Should it be `/usr/share/defaults/etc/scx_loader`, or `/usr/share/defaults/scx_loader`? Does it matter? We're really inconsistent about which pattern we use.

Reply:
Sounds like Aeryn is dropping the `etc/` in the path, so let's do that.